src/utils/utils.jl

```diff
@@ -297,7 +299,7 @@ function forward(sender::Tuple{Type, Symbol}, ::Type, method::Method)
     # Assert that function name always starts with `get_*`
     "`forward` only works for accessor methods that are defined as `get_*` or `set_*`"
     @assert startswith(string(method.name), r"set_|get_")
-    sender_type = "$(parentmodule(sender[1])).$(strip_module_name(sender[1]))"
+    sender_type = string(sender[1])
```

Comment:
Tell me if I’m missing something, but I think this should be sufficient.

Reply:
We can test it with the printing of the RTS